python/tvm/relay/frontend/tensorflow.py

```diff
@@ -484,6 +484,60 @@ def _impl(inputs, attr, params):
         return inputs[0]
     return _impl
 
+def _crop_and_resize():
+    def _impl(inputs, attr, params):
+        # input image is a 4-D tensor of shape [batch, image_height, image_width, depth]
+        # boxes is a 2-D tensor of shape [num_boxes, 4], 4 is for [y1, x1, y2, x2]
+        boxes = params.pop(inputs[1].name_hint).asnumpy()
+        data_shape = attr['_input_shapes'][inputs[0]]
+        data_dim = len(data_shape)
+        box_ind = params.pop(inputs[2].name_hint).asnumpy().tolist()
+        crop_size = params.pop(inputs[3].name_hint).asnumpy().tolist()
+        method = attr['method'].decode()
+
+        # 1) Crop
```

Comment:
Any reason why boxes at `0` handled differently ?

Reply:
handled all boxes in a loop now.